tooling/ci/install-required-packages-for-integration.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#
+#
+# Install all the necessary packages for spinning up a server
+#
+
+. ~/.bashrc
+
+sudo apt-get install libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+npm install --save-dev start-server-and-test
```

Comment:
Any reason we're using `npm` here vs `yarn`?

Reply:
the reason is that `yarn` doesn't work here..  saw some post online reported the similar issues..

Reply:
Boooooo ;p 

We should make a note in that file probably so no one tries to refactor it in a few months